exchanges/kraken/kraken_test.go

```diff
@@ -1355,3 +1359,31 @@ func TestWsCancelOrderJSON(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestGetHistoricCandles(t *testing.T) {
+	k.Verbose = true
```

Comment:
rm

Reply:
It was removed (and resolved) but added back in https://github.com/thrasher-corp/gocryptotrader/pull/479/commits/5f29b15caa36b838e8c15330808e3926f16103c2#diff-781dde07f68047deac7f84fb53dcb65eR1429 with further testing :)

Reply:
But I have removed it need to add a check for ".Verbose = true" in "_test.go" files as a githook <_<